lib/core/src/lib/form/components/widgets/core/form-field-validator.spec.ts

```diff
@@ -65,63 +64,72 @@ describe('FormFieldValidator', () => {
         });
 
-        it('should fail for dropdown with empty value', () => {
+        it('should fail (display error) for dropdown with empty value', () => {
             const field = new FormFieldModel(new FormModel(), {
                 type: FormFieldTypes.DROPDOWN,
-                value: '<empty>',
-                options: [{ id: 'empty', name: 'Choose option...' }],
+                value: null,
+                options: [
+                    { id: 'empty', name: 'Choose option...' },
+                    { id: 'none_empty', name: 'None empty' }
+                ],
                 hasEmptyValue: true,
                 required: true
             });
 
-            field.emptyOption = { id: '<empty>' } as FormFieldOption;
+            expect(field.value).toEqual({ id: 'empty', name: 'Choose option...' });
             expect(validator.validate(field)).toBeFalsy();
 
-            field.value = '<non-empty>';
+            field.value = { id: 'noneEmpty', name: 'None empty' };
             expect(validator.validate(field)).toBeTruthy();
         });
 
-        it('should fail for dropdown with zero selection', () => {
+        it('should fail (display error) for multiple type dropdown with zero selection', () => {
             const field = new FormFieldModel(new FormModel(), {
                 type: FormFieldTypes.DROPDOWN,
-                value: [],
-                hasEmptyValue: true,
+                value: [{ id: 'id_cat', name: 'Cat' }],
                 required: true,
-                selectionType: 'multiple'
+                selectionType: 'multiple',
+                hasEmptyValue: false,
+                options: [
+                    { id: 'id_cat', name: 'Cat' },
+                    { id: 'id_dog', name: 'Dog' }
+                ]
             });
 
-            field.emptyOption = { id: 'empty' } as FormFieldOption;
-            expect(validator.validate(field)).toBeFalsy();
-
+            const validateBeforeUnselect = validator.validate(field);
             field.value = [];
-            expect(validator.validate(field)).toBe(false);
+            const validateAfterUnselect = validator.validate(field);
+
+            expect(validateBeforeUnselect).toBeTruthy();
+            expect(validateAfterUnselect).toBeFalsy();
         });
 
-        it('should fail for dropdown with null value', () => {
+        it('should fail (display error) for dropdown with null value', () => {
             const field = new FormFieldModel(new FormModel(), {
                 type: FormFieldTypes.DROPDOWN,
                 value: null,
                 required: true,
-                options: [{ id: 'one', name: 'one' }],
-                selectionType: 'multiple'
+                options: [{ id: 'id_dog', name: 'Dog' }],
+                selectionType: 'multiple',
+                hasEmptyValue: false
             });
 
-            expect(validator.validate(field)).toBe(false);
+            expect(validator.validate(field)).toBeFalsy();
```

Comment:
are u sure about this ? 

toBe(false) checks if the value is strictly equal to false. It will only pass if the value is exactly false.
toBeFalsy() checks if the value is a falsy value. In JavaScript, falsy values include false, 0, '' (empty string), null, undefined, and NaN.

Reply:
Makes sense